docs/Node Operators/configuration-variables.md

```diff
@@ -2,811 +2,1355 @@
-  - [DATABASE_BACKUP_MODE](#database_backup_mode)
-  - [DATABASE_BACKUP_URL](#database_backup_url)
-  - [DATABASE_BACKUP_DIR](#database_backup_dir)
-  - [ETH_DISABLED](#eth_disabled)
-  - [EXPLORER_URL](#explorer_url)
-  - [EXPLORER_ACCESS_KEY](#explorer_access_key)
-  - [EXPLORER_SECRET](#explorer_secret)
-  - [TELEMETRY_INGRESS_URL](#telemetry_ingress_url)
-  - [TELEMETRY_INGRESS_SERVER_PUB_KEY](#telemetry_ingress_server_pub_key)
-  - [TELEMETRY_INGRESS_LOGGING](#telemetry_ingress_logging)
-  - [JSON_CONSOLE](#json_console)
-  - [LOG_LEVEL](#log_level)
-  - [LOG_SQL](#log_sql)
-  - [LOG_SQL_MIGRATIONS](#log_sql_migrations)
-  - [LOG_TO_DISK](#log_to_disk)
-  - [MIN_INCOMING_CONFIRMATIONS](#min_incoming_confirmations)
-  - [ETH_NONCE_AUTO_SYNC](#eth_nonce_auto_sync)
-  - [ETH_FINALITY_DEPTH](#eth_finality_depth)
-  - [ETH_HEAD_TRACKER_HISTORY_DEPTH](#eth_head_tracker_history_depth)
-  - [ETH_HEAD_TRACKER_MAX_BUFFER_SIZE](#eth_head_tracker_max_buffer_size)
-  - [ETH_HEAD_TRACKER_SAMPLING_INTERVAL](#eth_head_tracker_sampling_interval)
-  - [BLOCK_BACKFILL_DEPTH](#block_backfill_depth)
-  - [BLOCK_BACKFILL_SKIP](#block_backfill_skip)
-  - [ETH_LOG_BACKFILL_BATCH_SIZE](#eth_log_backfill_batch_size)
-  - [ETH_TX_REAPER_INTERVAL](#eth_tx_reaper_interval)
-  - [ETH_TX_REAPER_THRESHOLD](#eth_tx_reaper_threshold)
-  - [ETH_TX_RESEND_AFTER_THRESHOLD](#eth_tx_resend_after_threshold)
-  - [MINIMUM_CONTRACT_PAYMENT_LINK_JUELS](#minimum_contract_payment_link_juels)
-  - [MINIMUM_REQUEST_EXPIRATION](#minimum_request_expiration)
-  - [MINIMUM_SERVICE_DURATION](#minimum_service_duration)
-  - [OPERATOR_CONTRACT_ADDRESS](#operator_contract_address)
-  - [ORM_MAX_IDLE_CONNS](#orm_max_idle_conns)
-  - [ORM_MAX_OPEN_CONNS](#orm_max_open_conns)
-  - [ROOT](#root)
-  - [SECURE_COOKIES](#secure_cookies)
-  - [SESSION_TIMEOUT](#session_timeout)
-- [Advanced](#advanced)
-  - [ETH_MAX_IN_FLIGHT_TRANSACTIONS](#eth_max_in_flight_transactions)
-  - [ETH_MAX_QUEUED_TRANSACTIONS](#eth_max_queued_transactions)
-  - [DEFAULT_HTTP_TIMEOUT](#default_http_timeout)
-- [Misc notes](#misc-notes)
+  - [INSECURE_SKIP_VERIFY](#insecure_skip_verify)
+- [Notes on setting environment variables](#notes-on-setting-environment-variables)
+
+## Essential environment variables
+
+These are the only environment variables that are _required_ for a Chainlink node to run.
+
+### DATABASE_URL
+
+**Required**
+
+- Default: _none_
+
+The PostgreSQL URI to connect to your database. Chainlink nodes require Postgres versions >= 11. See the [Running a Chainlink Node](../running-a-chainlink-node/#set-the-remote-database_url-config) for an example.
+
+## General Node Configuration
+
+### CHAIN_TYPE
+
+- Default: _none_
+
+CHAIN_TYPE overrides all chains and forces them to act as a particular chain type. An up-to-date list of chain types is given in [`chaintype.go`](https://github.com/smartcontractkit/chainlink/blob/a710fcb3c1f3501b8df486b45c14423aefdb0876/core/chains/chaintype.go).
```

Comment:
```
[`chaintype.go`](https://github.com/smartcontractkit/chainlink/blob/a710fcb3c1f3501b8df486b45c14423aefdb0876/core/chains/chaintype.go).
```

Should we point this to a tag or a branch instead?

Reply:
Before merge, I'll update this to https://github.com/smartcontractkit/chainlink/blob/v1.1.0/core/chains/chaintype.go.